_ml-commons-plugin/ml-dashboard.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: default
+title: Managing ML models in OpenSearch Dashboards
+nav_order: 120
+---
+
+Machine Learning in OpenSearch dashboards is an experimental feature released in OpenSearch 2.6. It can't be used in a production environment. For updates on the feature’s progress or to leave feedback on improving the feature, see the [OpenSearch Forum discussion](https://forum.opensearch.org/t/feedback-ml-commons-ml-model-health-dashboard-for-admins-experimental-release/12494).
+{: .warning }
+
+Administrators of machine-learning (ML) clusters can use OpenSearch dashboards to manage and check the status of ML models running inside their cluster. This can aid in assisting ML developers with provisioning nodes to make sure their models run efficiently.
+
+As of OpenSearch 2.6, you can only upload models using the API. For more information about how to upload a model to your cluster, see [Upload model to OpenSearch]({{site.url}}{{site.baseurl}}/ml-commons-plugin/model-serving-framework#upload-model-to-opensearch)
+
+## Accessing Machine Learning in Dashboards
```

Comment:
User should add `ml_commons_dashboards.enabled: true` to `config/opensearch_dashboards.yml` to enable ML admin console, otherwise user can't see it.